scene/audio/audio_stream_player.cpp

```diff
@@ -32,6 +32,7 @@
 
 #include "core/config/engine.h"
 #include "core/math/audio_frame.h"
+#include "core/object/object.h"
```

Comment:
```suggestion
```
Added by IDE I assume, make sure to remove these or deactivate the feature, this is already included (or you couldn't have `AudioStreamPlayer`)

Reply:
Yes added by IDE, I will take a look, thanks !